Sources/Testing/CMakeLists.txt

```diff
@@ -97,16 +97,9 @@ target_link_libraries(Testing PRIVATE
 add_dependencies(Testing
   TestingMacros)
 target_compile_options(Testing PRIVATE
-  -enable-library-evolution)
+  -enable-library-evolution
+  -emit-module-interface -emit-module-interface-path $<TARGET_PROPERTY:Testing,Swift_MODULE_DIRECTORY>/Testing.swiftinterface)
```

Comment:
Maybe we might want to stop emitting `.swiftinterface` unless it's macOS because `.swiftinterface` is not installed and useless.